test/Riok.Mapperly.Tests/Mapping/ObjectPropertyUseNamedMappingTest.cs

```diff
@@ -683,4 +683,227 @@ public record B(int Value);
             .HaveDiagnostic(DiagnosticDescriptors.ReferencedMappingNotFound, "The referenced mapping named MapValue was not found")
             .HaveAssertedAllDiagnostics();
     }
+
+    [Fact]
+    public Task ShouldSupportExternalMappingsOnStaticMapping()
+    {
+        var source = TestSourceBuilder.MapperWithBodyAndTypes(
+            """
+            [MapProperty(nameof(A.Value), nameof(B.Value), Use = nameof(@OtherMapper.ModifyString)]
+            private static partial B Map(A source);
+            """,
+            "record A(string Value);",
+            "record B(string Value);",
+            """
+            class OtherMapper
+            {
+                public static string ModifyString(string source) => source + "-externally-modified";
+            }
+            """
+        );
+
+        return TestHelper.VerifyGenerator(source);
+    }
+
+    [Fact]
+    public Task ShouldSupportExternalMappingsOnStringWithSameNamespace()
+    {
+        var source = TestSourceBuilder.MapperWithBodyAndTypes(
+            """
+            [MapProperty(nameof(A.Value), nameof(B.Value), Use = $"{nameof(OtherMapper)}.{nameof(OtherMapper.ModifyString)}")]
+            private static partial B Map(A source);
+            """,
+            new TestSourceBuilderOptions { Namespace = "MyNamespace" },
+            "record A(string Value);",
+            "record B(string Value);",
+            """
+            class OtherMapper
+            {
+                public static string ModifyString(string source) => source + "-externally-modified";
+            }
+            """
+        );
+
+        return TestHelper.VerifyGenerator(source);
+    }
+
+    [Fact]
+    public Task ShouldSupportExternalMappingsOnStringWithUsingNamespace()
+    {
+        var source = TestSourceBuilder.MapperWithBodyInBlockScopedNamespace(
+            """
+            [MapProperty(nameof(A.Value), nameof(B.Value), Use = $"{nameof(OtherMapper)}.{nameof(OtherMapper.ModifyString)}")]
+            private static partial B Map(A source);
+            """,
+            new TestSourceBuilderOptions { AdditionalUsings = ["OtherNamespace"] }
+        );
+        source = TestSourceBuilder.Append(source, ["record A(string Value);", "record B(string Value);"]);
+
+        source = TestSourceBuilder.Append(
+            source,
+            "OtherNamespace",
+            [
+                """
+                class OtherMapper
+                {
+                    public static string ModifyString(string source) => source + "-externally-modified";
+                }
+                """,
+            ]
+        );
+
+        return TestHelper.VerifyGenerator(source);
+    }
+
+    [Fact]
+    public Task ShouldSupportExternalMappingsOnStringWithFullNameSpecification()
```

Comment:
A little bit overgrown, the test cases, and it needs to be duplicated if I wish to test every case on every attribute (MapProperty, MapValue, MapPropertyFromSource, IncludeMapping). Are you ok with that, or should I use a different approach?